src/test/java/org/opentripplanner/routing/graph/GraphSerializationTest.java

```diff
@@ -79,7 +95,6 @@ public void compareGraphToItself() {
       .getCachedPortlandGraph()
       .index();
     Graph originalGraph = cachedPortlandGraph.graph();
-    TransitModel originalTransitModel = cachedPortlandGraph.transitModel();
```

Comment:
Should we have a test, where we compare the transit models?

Reply:
We should. This PR however doesn't change the current test at all. This variable was just totally unused.

Reply:
Actually, a bit further down there is a test that compares the transit model.